test/recipes/30-test_evp_data/evpciph_aes_common.txt

```diff
@@ -240,6 +240,7 @@ Operation = ENCRYPT
 Plaintext = 6BC1BEE22E409F96E93D7E117393172A
 Ciphertext = 3B3FD92EB72DAD20333449F8E83CFB4A
 NextIV = 3b3fd92eb72dad20333449f8e83cfb4a
+InvertInit = 1
```

Comment:
Would it make sense to have the same test cases with and without InvertInit? Or just put the inverted initialization test to one of the evp_extra tests? It does not seem to me to be necessary to have so many test cases where one should trigger the failure already. Or am I wrong?

Reply:
Removed this commit and added some tests to evp_extra_test to test more precisely the errors recently encountered on s390x.  Please have a look at them.